test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -2726,10 +2777,9 @@ TEST_F(OobBackendMetricTest, Basic) {
   // Note that the server may send a new report while we're updating these,
   // so we set them in reverse order, so that we know we'll get all new
   // data once we see a report with the new CPU utilization value.
-  servers_[0]->orca_service_.SetNamedUtilization(kMetricName, 0.7);
```

Comment:
Still need to restore the original code here for setting the named metric.

Reply:
Updated, but I found this part of test was never working (no response in 5 attemps), confirmed with HEAD. I'll fix this in my next commit.

Reply:
Done

Reply:
Thanks for catching this!